### PR TITLE
DOC: update virtual box "getting started" documents

### LINKED ISSUE #1900: Update QIIME virtual box documentation
Hi--I am new to QIIME and have recently been spending oodles of time with the documentation :) , starting with that associated with the QIIME virtual box. In the process, I noticed a few typos and points where it appears some updates/corrections could help.  I have annotated the relevant files; since GitHub won't let me upload non-image files, I've put them on dropbox at the following links: 

1.Welcome.txt file notes: https://www.dropbox.com/s/fgb92hxdg230p7s/1.Welcome.txt%20notes.pdf?dl=0
3.Getting_started_with_QIIME.txt notes: https://www.dropbox.com/s/l8mqked2beik2z6/3.Getting_started_with_QIIME.txt.pdf?dl=0

Thanks for putting all this documentation together in the first place :)
Amanda

### PR DESCRIPTION
**Pull request by jairideout**
Updated the virtual box "getting started" documents that are included in `Before_you_start/`.

Fixes #1900.

@gregcaporaso can you please review?

### CODE REVIEW

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1592/

File: doc/vb_files/Before_you_start/1.Welcome.txt [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
+1. Release the mouse pointer from this window using either the left Command-key (Mac) or the right Control-key (Windows).
 
-Again, the directory name VBOXADDITIONS... may differ depending on your version of VirtualBox. You can run 'ls /media' to find the name of this directory. When prompted, enter the password: qiime
+2. Go to the VirtualBox menu and select Devices->Install Guest Additions... Click back into the Virtual Box, and then click 'Places' on the menu bar. Select VBOXADDITIONS_4.0.2_69518. (Note the name VBOXADDITIONS_4.0.2_69518 may differ depending on your version of VirtualBox.) A new window should appear in the virtual machine. Hit Open Autorun Prompt.
 
-4. When that command is finished running (i.e., you get a command prompt back), restart the virtual machine by clicking the power button to the right of word 'qiime' in the upper right corner of the virtual machine window and then selecting 'Restart...'.
-restart option can be under the 'qiime' box or under the power button icon to the right of the 'qiime' box).
+3. Open a terminal by clicking on the black window with a > in the top menu bar of the virtual machine, and paste this command into the terminal:
```
Comment (gregcaporaso) [inline doc/vb_files/Before_you_start/1.Welcome.txt]: *top menu bar* -> *side menu bar*

File: doc/vb_files/Before_you_start/1.Welcome.txt [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 
-After the virtual box restarts, if you are interested in knowing which software has been installed in this Virtual Machine move on to Document 2; if you are having problems with the network, i.e. the wget commands do not work, move to Document 3; otherwise move along to Document 4. All the Documents are in the Before_you_start folder on your desktop.
+  sudo /media/VBOXADDITIONS_4.0.2_69518/VBoxLinuxAdditions.run
+
+Again, the directory name VBOXADDITIONS_4.0.2_69518 may differ depending on your version of VirtualBox. You can run 'ls /media' to find the name of this directory. When prompted, enter the password: qiime
+
+4. When that command is finished running (i.e., you get a command prompt back), restart the virtual machine by clicking the power button to the right of the word 'qiime' in the upper right corner of the virtual machine window and then selecting 'Restart...'. The restart option can be under the 'qiime' box or under the power button icon to the right of the 'qiime' box.
```
Comment (gregcaporaso) [inline doc/vb_files/Before_you_start/1.Welcome.txt]: *the word 'qiime'* -> *'the words 'Qiime User'*

**Comment by gregcaporaso** [comment]
A couple of minor edits @jairideout - these are due to differences in the ubuntu version from when these were originally written. Thanks for taking care of this!

**Comment by jairideout** [comment]
Thanks for catching this! I made a few more edits to better reflect the GUI in Ubuntu 12.04.

**Comment by qiime-r2d2** [comment]
Build results will soon be (or already are) available at: http://ci.qiime.org/job/qiime-github-pr/1593/

**Pull request merged**